src/runtime_src/core/common/info_aie.cpp

```diff
@@ -420,14 +420,16 @@ populate_aie_shim(const xrt_core::device *device, const std::string& desc)
         ....
 */
 
-// This funtion populate a specific AIE core given as an input of [row:col]
+// This function populate a specific AIE core given as an input of [row:col]
 void
-populate_aie_core(const boost::property_tree::ptree& pt_core, boost::property_tree::ptree& tile,
-		  int row, int col)
+populate_aie_core(const boost::property_tree::ptree& pt_core, boost::property_tree::ptree& tile)
 {
   try {
     boost::property_tree::ptree pt;
     boost::property_tree::ptree empty_pt;
+
+    int row = tile.get<int>("row");
+    int col = tile.get<int>("column");
```

Comment:
More typing yes, but don't repeat the type, it's prone to error.
```suggestion
    auto row = tile.get<int>("row");
    auto col = tile.get<int>("column");
```